internal/app/orchestrator/orchestrator.go

```diff
@@ -143,7 +139,9 @@ func (o *orchestrator) CreateWatch(req transport.Request) (transport.Watch, func
 
 		// TODO (https://github.com/envoyproxy/xds-relay/issues/56)
 		// Support unnaggregated keys.
-		return o.downstreamResponseMap.delete(req), nil
+		// nil is a kill pill which will cause go-control-plane to error out the stream
+		_ = o.send(nil, watch, aggregatedKey)
```

Comment:
Would just be explicit here and do `_ = watch.Send(nil)`, since if `Send` errors it implies a fanout problem.

Reply:
`o.send` is just a wrapper over `watch.Send`. We do good amount of logging and metrics on channel full and I didn't want to repeat it all over in orchestrator